introduction.tex

```diff
@@ -317,92 +334,108 @@ \subsection{Capabilities required for modeling transition scenarios}
-		fuel cycle transition scenarios. Reproduced from Brown et al. \cite{brown_identification_2016}}
-	\label{tab:ffh}
-	\begin{tabularx}{\textwidth}{lXl}
-		\hline
-		\textbf{Functionality}&\textbf{Feature} &\textbf{Hierarchy} \\
-		\hline
-		\multirow{6}{*}{\shortstack{Composition \\ Features}} & Modeling of implicit consideration of fuel materials including primary fissile and fertile actinide isotopes & Basic \\
-		& Fuel's initial heavy metal mass modeled as lumped masses  of the remaining actinides and fission products to conserve mass & Basic \\
-		& Isotopic decay of materials in storage & Exemplary \\
-		& Modeling of intermediate isotopes (e.g. Pa-233) & Exemplary \\
-		& Tracking of fission products beyond a simple lumped sum & Exemplary \\
-		& Modeling of compounding materials in fuels and waste forms & Exemplary\\
-		
-		\hline
-		
-		\multirow{3}{*}{\shortstack{Fuel Cycle \\ Facility \\ Deployment}} & Facility deployment and retirement & Basic \\
-		& Construction time delays & Basic \\
-		& Strategic deployment to meet demand &Integral \\
-		
-		\hline
-		
-		\multirow{6}{*}{\shortstack{Front-end \\ Facilities}} & Source (mining and milling) & Basic \\
-		& Details of mines and mills including annual and total quantities available & Exemplary \\
-		& Conversion and enrichment facilities & Basic \\
-		& Timing and capacity of recycle facilities & Basic \\
-		& Fuel fabrication & Basic \\
-		& Time delays and losses in separations and fabrication & Basic \\
-		
-		\hline
-		
-		\multirow{4}{*}{\shortstack{Separations \\ and \\ Material \\ Recycling}} & Separations facilities may be required for \gls{UNF} & Basic \\
-		& Cooling time & Basic \\
-		& Losses in separations & Basic \\
-		& Material selection from the \gls{UNF} supply & Basic \\
-		
-		\hline
-		
-		\multirow{6}{*}{\shortstack{Reactor \\ Facility}} & Fueling: number of batches, cycle length and fuel per batch & Basic \\
-		& Multiple fuel types in reactor facility (driver, blanket) & Basic \\
-		& Pre-generated charge and discharge isotopic compositions & Basic \\
-		& Real time calculations based on reactor physics models & Exemplary \\
-		& Reactor facility lifetime, construction time, and decommissioning time & Basic \\
-		& Initial charge for first core and discharge for final core & Basic \\
-		
-		\hline
-	\end{tabularx}
-\end{table}
-\pagebreak
+    \centering
+    \caption {Nine common functionalities identified for \gls{NFCS} to perform
+                fuel cycle transition scenarios. Reproduced from Brown et al. \cite{brown_identification_2016}}
+    \label{tab:ffh}
+    \begin{tabularx}{\textwidth}{lXl}
+        \hline
+        \textbf{Functionality}&\textbf{Feature} &\textbf{Hierarchy} \\
+        \hline
+        \multirow{6}{*}{\shortstack{Composition \\ Features}} & Modeling of implicit consideration of fuel materials including primary fissile and fertile actinide isotopes & Basic \\
```

Comment:
What does "Modeling of implicit consideration of fuel... " mean? Maybe you just mean "Explicit modeling of fuel ... "